.semaphore/semaphore.yml

```diff
@@ -34,10 +35,15 @@ blocks:
         - name: "Install"
           commands:
             - cache restore
+            - tar zcvf cache.tgz .m2 target
```

Comment:
at this stage we are only interested in the .m2 cache to avoid downloading all the dependencies again.

Reply:
refactored per suggestions